javascript/javascript2/week1/js2w1hw-hyf.js

```diff
@@ -0,0 +1,105 @@
+//Write a function that finds the shortest word of an array of words ex---
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function shortestWords(danishWords) {
+let shortValue = danishWords[0];
+   for(const shortWord of danishWords){
+     if(shortValue.length > shortWord.length){
+       shortValue = shortWord;
+     }
+   }
+   return shortValue
+  }
+console.log("Shortest word from string array = "+ shortestWords(danishWords));
+
```

Comment:
I am a little bit confused with the names of variables `shortValue` and `shortWord`, because for me it's difficult to grasp the difference.

Reply:
ohhk :( ..from next time will take care of it..